doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -60,6 +79,545 @@ Recommended changes
 
 The following changes are recommended for your application to work optimally after the migration.
 
+* For the Matter samples and applications
+
+  * The new API and helper modules have been added to the :file:`samples/matter/common` directory.
+    All Matter samples and applications have been changed to start using the common software modules.
+    You can follow the new approach and migrate your application to use the common software modules, what will signicantly reduce the size of a code required to be implemented in the application.
```

Comment:
```suggestion
    You can follow the new approach and migrate your application to use the common software modules.
    This will significantly reduce the size of code required to be implemented in the application.
```